src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -144,4 +168,59 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             _OpenJsonHandlers(nullptr, target);
         }
     }
+
+    void MainPage::_InitializeProfilesList()
+    {
+        // Manually create a NavigationViewItem for each profile
+        // and keep a reference to them in a map so that we
+        // can easily modify the correct one when the associated
+        // profile changes.
+        for (const auto& profile : _settingsSource.AllProfiles())
+        {
+            auto navItem = _CreateProfileNavViewItem(profile);
+            SettingsNav().MenuItems().Append(navItem);
+        }
+
+        // Top off (the end of the nav view) with the Add Profile item
+        MUX::Controls::NavigationViewItem addProfileItem;
+        addProfileItem.Content(box_value(RS_(L"Nav_AddNewProfile/Content")));
+        addProfileItem.Tag(box_value(L"AddProfile"));
+        addProfileItem.SelectsOnInvoked(false);
+
+        FontIcon icon;
+        // This is the "Add" symbol
+        icon.Glyph(L"\xE710");
+        addProfileItem.Icon(icon);
+
+        SettingsNav().MenuItems().Append(addProfileItem);
+    }
+
+    void MainPage::_CreateAndNavigateToNewProfile(const uint32_t index)
+    {
+        auto newProfile = _settingsSource.CreateNewProfile();

```

Comment:
```suggestion
        auto newProfile = Settings().CreateNewProfile();
```
At some point, we'll be doing all of this off of the clone. So referencing `Settings()` will be a nice way to make that transition easier.

Reply:
ooh yes true, i'll change all my occurrences